examples/subnet-tags/index.ts

```diff
@@ -18,6 +18,7 @@ const vpc = new awsx.ec2.Vpc("vpc",
             {type: SubnetType.Public, tags: {"kubernetes.io/role/elb": "1", ...tags}},
             {type: SubnetType.Private, tags: {"kubernetes.io/role/internal-elb": "1", ...tags}},
         ],
+        subnetStrategy: "Legacy", // Upstream awsx default is being switched to "Auto", explicitly set to "Legacy".
```

Comment:
Should we maybe switch this to auto? Having legacy in the examples might be a bit discouraging

Reply:
Good idea! I was just throwing up something quick in the draft PR to get this working.